apps/tailwind-components/components/form/Fields.vue

```diff
@@ -59,13 +59,20 @@ onMounted(() => {
   <div ref="container">
     <template v-for="column in columns">
       <div
-        v-if="column.columnType === 'HEADING'"
+        v-if="
+          column.columnType === 'HEADING' || column.columnType === 'SECTION'
+        "
         :id="`${column.id}-form-field`"
         v-intersection-observer="[onIntersectionObserver, { root: container }]"
       >
         <h2
-          class="first:pt-0 pt-10 font-display md:text-heading-5xl text-heading-5xl text-form-header pb-8"
-          v-if="column.label"
+          class="first:pt-0 pt-10 font-display text-form-header pb-8 scroll-mt-4"
```

Comment:
The heading level might need to be adjusted. The form title should have the highest level (e.g., h2) and sections should come next (h3). In context of the form, if the form/modal title has an h2 element, then it should be h3.